01_GettingStartedWithDataLab/02_GettingStartedWithDataLab.ipynb

```diff
@@ -130,26 +129,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "'robertdemo'"
+       "'ajacques'"
       ]
      },
-     "execution_count": 3,
+     "execution_count": 12,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# As an anonymous user, you can issue queries to the database or retrieve image cutouts, but\n",
-    "# not store your results in virtual storage or myDB\n",
-    "\n",
-    "# Get token for an authenticated user if you wish to use virtual storage or myDB\n",
```

Comment:
Good to remove the sentence about anonymous access.
For the authenticated access, please have this sentence in the commented-out login cell: "If you need to log in to Data Lab:" (and then the code line).